lib/cog/adapters/slack/api.ex

```diff
@@ -276,8 +276,9 @@ defmodule Cog.Adapters.Slack.API do
   defp cache_item(%{"is_group" => true, "id" => id, "name" => name}),
     do: %{id: id, name: name}
   # users have names, but not "is_user" fields :( they do have profiles, though
-  defp cache_item(%{"id" => id, "name" => name, "profile" => %{}}),
-    do: %{id: id, handle: name}
+  defp cache_item(%{"id" => id, "name" => name, "profile" => profile}),
+  do: %{id: id, handle: name, email: profile["email"], first_name: profile["first_name"],
+        last_name: profile["last_name"]}
```

Comment:
This is fine for now since we're focusing on Slack, but this feature won't work in HipChat or IRC until those adapters are adjusted.

We should make issues tracking both of those so we don't lose track.